app/Transformers/ProductTransformer.php

```diff
@@ -335,4 +395,95 @@ private function removeColorOption(array $variant, ?int $colorPosition): array
             ...$updatedOptions,
         ];
     }
+
+    private function checkVariantMigrationEligibility(array $data, Collection $variants): bool
+    {
+        if(empty($data['product_links'])) {
+            return false;
+        }
+
+        if (!in_array(MigrationProduct::MATADOR_WEBSITE_ID, $data['extension_attributes']['website_ids'], true)) {
```

Comment:
:large_blue_circle: If possible, I recommend using named arguments to avoid the [boolean trap](https://ariya.io/2011/08/hall-of-api-shame-boolean-trap).

```suggestion
        if (!in_array(MigrationProduct::MATADOR_WEBSITE_ID, $data['extension_attributes']['website_ids'], strict: true)) {
```